libkbfs/folder_branch_ops.go

```diff
@@ -3612,6 +3611,7 @@ func (fbo *folderBranchOps) undoMDUpdatesLocked(ctx context.Context,
 			fbo.notifyOneOpLocked(ctx, lState, io, rmd)
 		}
 	}
+	// TODO: update the edit history?
```

Comment:
@jzila: You might be undoing file edits/creates, which have already been added to the history.  So in theory those need to be undone in the history as you undo them on the file system.